pkg/policy/id.go

```diff
@@ -34,7 +32,7 @@ func (id ID) IsValid() bool {
 		return true
 	}
 
-	return idRegexp.MatchString(string(id))
+	return utf8.ValidString(string(id))
```

Comment:
According to https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html, `ID`  can be any arbitrary string.  This function needs to be just one liner `return utf8.ValidString(string(id))`

Reply:
okay